### PR TITLE
[it] validation-inline

### PR DESCRIPTION
**Pull request by masterix21**

### CODE REVIEW

File: src/it/validation-inline.php [hunk 1]
```diff
@@ -13,110 +13,110 @@
...
-    'string'               => 'This must be a string.',
-    'timezone'             => 'This must be a valid zone.',
-    'unique'               => 'This has already been taken.',
-    'uploaded'             => 'This failed to upload.',
-    'url'                  => 'This format is invalid.',
-    'uuid'                 => 'This must be a valid UUID.',
+    'starts_with'          => 'Deve iniziare con uno dei seguenti: :values.',
+    'string'               => 'Deve essere una stringa.',
+    'timezone'             => 'Deve essere una zona valida.',
+    'unique'               => 'E\' già in uso.',
```
Comment (caouecs) [inline src/it/validation-inline.php:116]: Use " to not have \\'       "E' già in uso"

**Review by caouecs — CHANGES_REQUESTED** [review]
Grazie but just a problem

**Comment by masterix21** [comment]
Done. 

If you like, I could be a referent for the Italian language. I'm working on the other translations.

**Comment by caouecs** [comment]
Grazie for the translations and your proposition to be referent ( we come back fast to you )

**Pull request merged**